<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainForm</class>
 <widget class="QWidget" name="mainForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitPy</string>
  </property>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать репозиторий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commitButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>89</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newbranchButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>89</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Новая ветка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Закончить работу.</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="logEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addfilesButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить фалы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delfilesButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить файлы</string>
   </property>
  </widget>
  <widget class="QTreeView" name="branchtreeView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ветки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Лог</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
